matchms/filtering/__init__.py

```diff
@@ -26,6 +29,12 @@
 
     Maximum intensity is 1.00
 
+.. image:: ../_static/filtering_sketch.png
```

Comment:
Caption only supported by figure not by image directive.
```suggestion
.. figure:: ../_static/filtering_sketch.png
```

Reply:
Oh, of course. Thanks!